synapse/lib/cell.py

```diff
@@ -206,6 +207,14 @@ async def __anit__(self, cell, link, user):
     async def initCellApi(self):
         pass
 
+    @adminapi()
```

Comment:
Lets do `log=True` here so we can log the caller ( important in the case of a non-root telepath session to know who is freezing the cell ).